types/fusebit__add-on-sdk/index.d.ts

```diff
@@ -0,0 +1,110 @@
+// Type definitions for @fusebit/add-on-sdk 3.1
+// Project: http://fusebit.io/
+// Definitions by: Andrew Miller <https://github.com/andrewrmiller>
+//                 Trent Hashimoto <https://github.com/hashiphoto>
+//                 Andy Dam <https://github.com/andydam>
```

Comment:
```suggestion
```
you also need ACK from this person, or convert to `@credit` mention. The DT package owners receive automated notification, so we expect explicit opt-in (ACK) (at least recently)

Reply:
Andy has approved this PR.  Is that sufficient?